### ISSUE TITLE
Can we Pretty print the code generated in playground ?

### ISSUE THREAD

**Issue by vishnukool** [langchain-ai/langserve#598]
It looks really ugly and is a very easy change to pretty print on code and other formatting 

<img width="771" alt="Screenshot 2024-04-09 at 1 02 20 AM" src="https://github.com/langchain-ai/langserve/assets/857006/04736564-b208-4818-afe1-8b793f1d6893">

**Comment by eyurtsev** [comment]
Yep this is a reasonable enhancement for the playground. We're still scoping out where to focus more in langserve (playground or more deployment options). Let me know if you have any other feedback

**Comment by vishnukool** [comment]
The biggest other feedback is that the playGround doesn't seem to load chatHIstory automatically. If in the old playground I have a conversation, you'd expect the chatwidget to auto populate the history. But that is not the case. 

I know there's a new playground chat experience, but that currently has a lot of limitations in being able to do scratch pad etc. So just having a chat widget in old playground that can auto populate in memory chat history conversation would be great!
